src/common/Grid.js

```diff
@@ -337,13 +337,17 @@ var Grid = fc.Grid = RowRenderer.extend({
 
 	// Binds DOM handlers to elements that reside outside the grid, such as the document
 	bindGlobalHandlers: function() {
-		$(document).on('dragstart sortstart', this.externalDragStartProxy); // jqui
+		if (this.view.opt('droppable')) {
```

Comment:
can the option be changed at runtime? if so, do we rebind global handlers when that happens?